exercise-book/src/iterators.md

```diff
@@ -0,0 +1,324 @@
+# Iterators
+
+In this exercise, you will learn to manipulate and chain iterators. Iterators are a functional way to write loops and control flow logic.
+
+## After completing this exercise you are able to
+
+- write a Rust iterator
```

Comment:
Should be re-worded. We don't teach them implementing `Iterator` trait. I call them "iterator chains", but maybe there's a better term for that.

Reply:
Good call, rephrased.